libmscore/read301.cpp

```diff
@@ -297,16 +297,7 @@ bool MasterScore::read(XmlReader& e)
 
 void MasterScore::addMovement(MasterScore* score)
       {
-      score->_movements = _movements;
       _movements->push_back(score);
```

Comment:
This will break compatibility with the old albums? That's fine as long it is intentional.

Reply:
It does not break backwards-compatibility.

Reply:
:+1: